doc/connectivity/parcellating.rst

```diff
@@ -19,8 +19,8 @@ into homogeneous regions from functional imaging data.
     <http://journal.frontiersin.org/article/10.3389/fnins.2014.00167/full>`_
     Frontiers in neuroscience 8.167 (2014): 13.
 
-Data loading and massaging
-===========================
+Data loading
+============
 
 Resting-state data
 -------------------
```

Comment:
Now that you've simplified this example (which is a great thing), you could merge this header with the one above: there is no need for a subsection.